polling_stations/apps/file_uploads/models.py

```diff
@@ -54,6 +55,7 @@ class Upload(models.Model):
 
     objects = UploadQuerySet.as_manager()
     warning_about_pending_sent = models.BooleanField(default=False)
+    upload_user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
```

Comment:
Drive by... Do we definitely want to `CASCADE` the deletes here. I'm not sure - in some ways a council 'owns' the upload object more than the user. I think perhaps `SET_NULL` would be more appropriate - but would be good to know what you think.

Reply:
I agree, this needs a change. I think if we change to `SET_NULL` then we should add a `Upload.upload_user.email` check where we are sending emails, right?